UnitTests/ChartTypeSelectionTests/GroupVerticalBarChartTests.cs

```diff
@@ -1,4 +1,6 @@
-﻿using NUnit.Framework;
+﻿using Microsoft.AspNetCore.Hosting.StaticWebAssets;
```

Comment:
This appears to be here by accident, doesn't seem it's needed

Reply:
System.Linq seems to be unused too a few lines below

Reply:
Removed the unused statements